### PR TITLE
Fix: design fixes on notifications screen

### PR DESCRIPTION
**Pull request by josepfo**

### CODE REVIEW

**Comment by josepfo** [comment]
[Design fixes on notifications screen](https://trello.com/c/JacvJ4OV/832-design-fixes-on-notifications-screen)

File: new-lamassu-admin/src/components/inputs/cashbox/Cashbox.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ const Cashbox = ({
   labelClassName,
   inFiatBalanceAlerts
 }) => {
-  const classes = cashboxClasses({ percent, cashOut })
+  const classes = cashboxClasses({ percent, cashOut, inFiatBalanceAlerts })
```
Comment (chaotixkilla) [inline new-lamassu-admin/src/components/inputs/cashbox/Cashbox.js]: I'm not really a fan of having a flag for a specific screen, as we might use that same behavior in the future, and the variable name shouldn't be tied to a specific screen. My suggestion is to call this flag something like `shouldApplyColorVariance`/`shouldApplyColorGradient` (I don't like the gradient one as much, as we're not working with gradients)

File: new-lamassu-admin/src/components/inputs/cashbox/Cashbox.styles.js [hunk 2]
```diff
@@ -11,8 +11,12 @@ const colors = {
   }
 }
 
-const colorPicker = ({ percent, cashOut }) =>
-  colors[cashOut ? 'cashOut' : 'cashIn'][percent >= 50 ? 'full' : 'empty']
+const colorPicker = ({ percent, cashOut, inFiatBalanceAlerts }) => {
```
Comment (chaotixkilla) [inline new-lamassu-admin/src/components/inputs/cashbox/Cashbox.styles.js]: This flag

File: new-lamassu-admin/src/components/inputs/cashbox/Cashbox.js (hunk 1 above)
Comment (chaotixkilla) [reply new-lamassu-admin/src/components/inputs/cashbox/Cashbox.js]: Also, default value should be `false` when this flag is not specified in props. You can make the default value `true`, but the flag would need to be called something as `dontApplyColorVariance`

File: new-lamassu-admin/src/pages/Notifications/sections/FiatBalanceAlerts.js [hunk 3]
```diff
@@ -58,29 +58,57 @@ const FiatBalance = ({ section, min = 0, max = 100, fieldWidth = 80 }) => {
...
+            <div className={classes.first}>
+              <div className={classes.row}>
+                <Cashbox
+                  labelClassName={classes.cashboxLabel}
+                  emptyPartClassName={classes.cashboxEmptyPart}
+                  percent={
+                    values.fillingPercentageCassette1 ??
+                    data?.fillingPercentageCassette1
+                  }
+                  inFiatBalanceAlerts={true}
```
Comment (chaotixkilla) [inline new-lamassu-admin/src/pages/Notifications/sections/FiatBalanceAlerts.js]: This here. Also, I believe you can get away with omitting `={true}`, and just call `applyColorVariant` or whatever name you decide better fits the flag's purpose

File: new-lamassu-admin/src/components/inputs/cashbox/Cashbox.js (hunk 1 above)
Comment (chaotixkilla) [reply new-lamassu-admin/src/components/inputs/cashbox/Cashbox.js]: Ah, noticed you're calling a pre-existing flag here. I guess you can ignore the variable name change

**Review by chaotixkilla — CHANGES_REQUESTED** [review]

**Review by RafaelTaranto — APPROVED** [review]

**Pull request merged**